### PR TITLE
Portfolio2

### PR DESCRIPTION
**Pull request by olyolyoly**

### CODE REVIEW

File: css/portfolio.css [hunk 1]
```diff
@@ -128,33 +129,20 @@ nav {
...
 
-.project-hover:hover {
-    position: absolute;
-    top: 0;
-    left: 0;
-    width: 100%;
-    height: 100%;
-    background: rgba(220, 0, 0, 0.5);
-    opacity: 0;
+/*.project-hover:hover {*/
```
Comment (neilbulloch) [inline css/portfolio.css:132]: You have commented out this CSS, if you don't need it you should probably delete it

File: index.php [hunk 2]
```diff
@@ -23,7 +33,7 @@
     <section id="home">
         <div class="image-container">
             <div class="image-title">
-                <h1 class="title-style">Oliver Matthews</h1>
+                <h1 class="title-style"><?php echo 'Oliver Matthoews'; ?></h1>
```
Comment (neilbulloch) [inline index.php:36]: how come you are using php to echo out the name? Why not have it as it was?

**Review by neilbulloch — COMMENTED** [review]
It looks good ;)

File: css/portfolio.css (hunk 1 above)
Comment (olyolyoly) [reply css/portfolio.css:132]: actioned

File: index.php [hunk 3]
```diff
@@ -63,9 +73,11 @@ <h4>About</h4>
             <h4>Portfolio</h4>
         </div>
             <div class="portfolio">
-                <div class="projects project1 project-hover">Example 1</div>
-                <div class="projects project2 project-hover">Example 2</div>
-                <div class="projects project3 project-hover">Example 3</div>
+                <?php
+                foreach ($result as $project) {
```
Comment (mporam) [inline index.php:77]: what if `$result` is empty?

File: portfolio.php [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<?php
```
Comment (mporam) [inline portfolio.php:1]: what is this file?

**Pull request merged**